disassembler/disassembler.py

```diff
@@ -0,0 +1,9 @@
+class Disassembler:
+    def configure(self, options):
+        raise NotImplementedError("configure")
```

Comment:
`options` here is a module (a plain file for the not python people).
splat has been usually just accessing module globally and not passing it around, so I suggest to just do that.
The alternative would be to pass a `SplatOpts` instead
```suggestion
    def configure(self, opts: SplatOpts):
        raise NotImplementedError("configure")
```